modules/owncloud_web/pages/web_for_users.adoc

```diff
@@ -159,6 +163,8 @@ image:web-for-users/spaces-menu.png[Spaces Menu,width=300]
 
 (1) ... By deactivating, a Manager can disable any access to a space including sharing and syncing. This can be used to keep data in a frozen state before finally deleting it later on. To regain full access including sharing and syncing, a deactivated space can be activated again.
 
+(2) ... When creating a "Quicklink" with the role `Internal` assigned, recipients of those links can quickly access a resource they already have access to. This makes it easier to reference content in big spaces, therefore the name Internal. To access a Quicklink, the user must have an account and a grant to that resource with one of the other roles available. If the share recipient has no account and has no grant to access the shared resource, accessing the linked content is not possible and nothing will be shown.
```

Comment:
```suggestion
(2) ... When creating a "Quicklink" with the role `Internal` assigned, recipients of those links can quickly access a resource they already have access to. This makes it easier to reference content in big spaces, therefore the name Internal. To access a Quicklink, the user must have an account and a grant to that resource with one of the other roles available. If the link recipient has no account and has no grant to access the shared resource, accessing the linked content is not possible and nothing will be shown.
```

I suppose that someone who creates a "Quicklink" then sends that URL to "someone else" by "some means". So it could be anyone who "receives" (is told) the Quicklink URL. That person could be called the "link recipient".